components/dashboard/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { defi, price } from "../../utils/tzktHooks";
+import Assets from "./assets";
+import Balance from "./balance";
+import Donut from "./donut";
+
+type DashboardProperty = {
```

Comment:
Idem pour Props

Reply:
fixed in: https://github.com/marigold-dev/tzsafe-ui/pull/167/commits/456e29db1154dddebeeac6f64d6e39a3de00d72d